### PR TITLE
Added centralizer component

### LINKED ISSUE #1: Create centralize area independent off hero section
The HeroSection is not infinite creating a white space after 1440px 

please make the hero infinite and a differente centralized area

### PR DESCRIPTION
**Pull request by ViniciusDaCunha**

### CODE REVIEW

File: src/components/Centralizer/Centralizer.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import "./Centralizer.css";
+
+type CentralizerParam = { children: React.ReactNode };
```
Comment (giolazzareschi) [inline src/components/Centralizer/Centralizer.tsx]: @ViniciusDaCunha  please use the plural names for the params like

CentralizerParams

File: src/components/Centralizer/Centralizer.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.Centralizer {
+  width: 1440px;
+  margin: auto 0;
```
Comment (giolazzareschi) [inline src/components/Centralizer/Centralizer.css]: @ViniciusDaCunha hey please uso margin: 0 auto to centralize horizontally

Comment (ViniciusDaCunha) [reply src/components/Centralizer/Centralizer.css]: ok

**Review by giolazzareschi — COMMENTED** [review]
code reviewed
tested locally

**Pull request merged**